CHANGELOG.md

```diff
@@ -62,6 +62,67 @@ You should have this:
 <p class="nhsuk-u-font-size-36">
 ```
 
+- Updated default `name` attributes for Date input component
+
+The default name attributes for the date input elements now use square brackets around the date part, for instance `dob[day]`, `dob[month]`, `dob[year]`. Previously they used hyphens (`dob-day`). 
+
+The square brackets mean that the date parts will get combined into an object when using the NHS Prototype Kit, like this:
+
+```json
+{
+  "dob": {
+    "day": "13",
+    "month": "12",
+    "year": "1984"
+  }  
+}
+```
+
+This allows the values to be more easily pre-filled, which you can now do by passing an object to the `values` key:
+
+```njk
+{{ dateInput({
+  namePrefix: "dob",
+  fieldset: {
+    legend: {
+      text: "What is your date of birth?"
+    }
+  },
+  values: data.dob
+}) }}  
+```
+
+You can override this new default by setting the `name` attribute on the individual date parts within `items`:
+
+```njk
+{{ dateInput({
+  fieldset: {
+    legend: {
+      text: "What is your date of birth?"
+    }
+  },
+  items: [
+    {
+      name: "dob-day",
+      label: "Day",
+      classes: "nhsuk-input--width-2"
+    },
+    {
+      name: "dob-month",
+      label: "Month",
+      classes: "nhsuk-input--width-2"
+    },
+    {
+      name: "dob-year",
+      label: "Year",
+      classes: "nhsuk-input--width-4"
+    }
+  ]
+}) }}
+```
+
+This change was made in [PR 994](https://github.com/nhsuk/nhsuk-frontend/pull/994).
```

Comment:
@anandamaryon1 think I've done this now, following the format in the first bullet (`- Updated back link and breadcrumbs ([PR 1002](https://github.com/nhsuk/nhsuk-frontend/pull/1002))`)

The middle one (`- Replaced font size class `nhsuk-u-font-size-32` with `nhsuk-u-font-size-36`, based on the new type scale`) doesn't link to the PR yet though?

Also I wonder if these should be sub-sub-headings instead of bullets?

Reply:
Thanks, that's it. 

Oh yes, good spot, I'll add it in on this branch/PR, to save making another PR for it.